doc/modules/multimap.md

```diff
@@ -67,6 +67,7 @@ Optional map configuration attributes:
 * `group` - group for the symbol (can be redefined in `metric`)
 * `require_symbols` - expression of symbols that have to match for a specific message: [learn more](#conditional-maps)
 * `filter` - match specific part of the input (for example, email domain): [here](#map-filters) is the complete definition of maps filters
+* `extract_from` - specifies the source (smtp or mime or both) from which to extract the sender or recipient address
```

Comment:
We need to specify that this attribute has meaning merely for `type=from` and `type=rcpt` maps.